widgets/pie/options.go

```diff
@@ -0,0 +1,50 @@
+package pie
+
+import (
+	"github.com/mum4k/termdash/cell"
+)
+
+// Option defines a function that sets a specific option for the Pie widget.
+type Option interface {
+	// set sets the provided option.
+	set(*options)
+}
+
+// option implements Option.
+type option func(*options)
+
+// set implements Option.set.
+func (o option) set(opts *options) {
+	o(opts)
+}
+
+// options stores the provided options.
+type options struct {
+	colors []cell.Color
+}
+
+// validates the provided options
+// at the moment no validation is performed cause options are not required
+func (o *options) validate() error {
+	return nil
+}
+
+// newOptions creates a new options instance.
+func newOptions() *options {
+	return &options{
+		colors: DefaultColors,
+	}
+}
+
+// DefaultColors defines a default set of colors used for rendering pie chart segments.
+// These colors are chosen from the predefined cell.Color constants and include a variety
+// of primary and secondary colors to ensure visual distinction between segments.
```

Comment:
Can we document whether the number of colors in the list is significant and how? E.g. what happens if there are more values than colors? Will they get reused in a round-robin fashion? Something else?